client/securedrop_client/gui/actions.py

```diff
@@ -74,26 +74,24 @@ def _set_enabled_initial_value(self) -> None:
 
 
 class DeleteSourceAction(QAction):
```

Comment:
(these changes aren't strictly needed and I might revert/squash them; I initially was thinking we could use a single DeleteSourceAction for both the source menu and the toolbar, but in the case of the source menu, the target sources are supplied when the action is created, but in the case of the toolbar, the action is created before the target sources are available.  The benefit to keeping the changes in is that they will work for if/when we _do_ eventually deprecate the source menu and consolidate around one deletion mechanism.)